### ISSUE TITLE
converse.js answers disco info wrong

### ISSUE THREAD

**Issue by lovetox** [conversejs/converse.js#1144]
I just received this from a converse.js user after issuing a disco info with the entity caps hash node
```
<iq xmlns="jabber:client" to="user1@domain1/gajim.G5BIBBZD" from="user@domain/converse.js-64190275" type="result" id="fb1d647b-6bc0-4b3f-a69a-3801abbacaff">
<identity type="web" name="Converse.js" category="client" />
<feature var="http://jabber.org/protocol/chatstates" />
<feature var="http://jabber.org/protocol/disco#info" />
<feature var="http://jabber.org/protocol/muc" />
<feature var="http://jabber.org/protocol/rosterx" />
<feature var="jabber:x:conference" />
<feature var="jabber:x:oob" />
<feature var="urn:xmpp:carbons:2" />
<feature var="urn:xmpp:http:upload:0" />
<feature var="urn:xmpp:mam:2" />
<feature var="urn:xmpp:message-correct:0" />
<feature var="urn:xmpp:spoiler:0" />
<feature var="urn:xmpp:xbosh" />
<feature var="vcard-temp" />
</iq>
```

query node is missing

**Comment by jcbrand** [comment]
Relevant section of the XEP: https://xmpp.org/extensions/xep-0115.html#howitworks

**Comment by jcbrand** [comment]
Fixed in #1145
